src/views/HomeView.vue

```diff
@@ -1,6 +1,66 @@
-<script setup>
-</script>
+<script setup></script>
 
 <template>
-  <h1 class="example">For Example</h1>
+  <main>
+    <h1 class="example" id="home">
+      Welcome to the Home page
+      <br />
+      <br />
+      <br />
+      <br />
+      <br />
+      <br />
+      <br />
+      <br />
+      <br />
+      <br />
+
```

Comment:
There is no need to add excessive \<br\>s especially when it might create a merge conflict for others. While I understand you were testing the anchor links, it's better to remove them now.

Reply:
Same with the rest of the file